community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/PhysicalWritableLogChannel.java

```diff
@@ -141,5 +163,6 @@ private ByteBuffer bufferWithGuaranteedSpace( int spaceInBytes ) throws IOExcept
     public void close() throws IOException
     {
         emptyBufferIntoChannelAndClearIt();
+        closed = true;
```

Comment:
If close can happen concurrently with writes, won't we have concurrent access to the buffer through the empty... method?